civis/civis-aqueduct-utils/civis_aqueduct_utils/share.py

```diff
@@ -2,8 +2,11 @@
 Small utility for creating shareable links to Civis services.
 """
 import argparse
-
+import base64
```

Comment:
Can we add this utility function to a new file, rather than putting it in `share.py`? Maybe `github.py`.